### DISCUSSION TITLE
Create new local user. How retrieve generated password ?

### DISCUSSION THREAD

**Discussion by Pavorjfo** [nilsteampassnet/TeamPass#3968] [Q&A] [answered]
Hi everyone,

Sorry to ask a simple question like that i cant create a local account :(. 
But i already use this pretty good project in previous version 2.X and that was possible.
Actually i need to create few "local" account no connection with any smtp server or ldap connection.
I have create an account only for me, regenerate OTP but when i try to connect with my user what kind of password i'm suppose to used ? (i try the otp regenerate but ofc didn't working).

How i can retrieve the pw auto generated in the db please ?

Info : 
 Current version loaded 3.1.0.9
 Alpine Linux v3.18

PHP version: 8.2.7
Memory limit: 128M
Memory usage: 525.51 KB
Maximum time execution: 120
Maximum file size upload: 100M

Server version: 10.5.23-MariaDB-1:10.5.23+maria~ubu2004
Database size: 0.95MB

**Comment by Pavorjfo** [comment]
Ok i read your documentation and your code, actually i work on a script to decrypt the local password who is generated auto for each user 

I still dont understand why we dont have anymore an option, only for administrator to see the generated password for a fresh creation of user and let the user change the password at first logon ???
Or it's just me who dont see where we can do that -_-...

**Comment by Pavorjfo** [comment]
Hello @nilsteampassnet , 
Je n'arrive pas à avoir un code propre pour décoder les mdp :(.
Du coup je génère moi même via un script le mdp et je l'injecte en base, ce qui n'est pas très propre.

C'est possible de remettre disponible la fonctionnalité de générer les mdp sans LDAP ni mail please ? :pray:

**Reply by nilsteampassnet** [reply]
@Pavorjfo
This was removed for code and cases simplification.
I could see how to put this back but it will be very basic.

**Reply by Pavorjfo** [reply]
Hi @nilsteampassnet 
If you can bring back this function should be perfect for us ! We need a teampass not linked with our LDAP or Mailing. 
No need something complex just a simple way to generate password for the local user and the possibility to see the password who is generated, and the user change it at is first logon.
Thanks alot @nilsteampassnet for your answer and your work !

**Reply by nilsteampassnet** [reply, marked as answer]
@Pavorjfo 
I provided a solution
![image](https://github.com/nilsteampassnet/TeamPass/assets/1197546/5bc01e94-a09a-49be-9f0f-7e8cbd24c992)

I hope this fit the expectation of Teampass instances full local without mail.

**Reply by Pavorjfo** [reply]
Hello @nilsteampassnet 
Just perfect !! Enough for us.
Again thanks a lot @nilsteampassnet !
